kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -348,18 +343,22 @@
     width: ($iphone-5-width - 20)px
 
   .terms
-    background-color: $core-bg-light
-    color: $core-text-annotation
-    height: 6em
-    overflow-y: scroll
-    padding: 0.5em
-    margin-bottom: 1em
-    p
-      margin-top: 0
+    height: 80vh
+    width: 80vw
+    &-agreement-checkbox
```

Comment:
As we mentioned we should move any instant schools specific styling to there?

Reply:
@indirectlylit @rtibbles 

I'm a little confused about the purpose of the separated styling and feature branches. 

Christian is saying that the styles are going to be used to port styles to logo and color work to both instant schools and network schools and that, following this logic, that the style branch should not contain any instant-schools-specific styles like ToS.

This makes sense to me, but my understanding was that _all_ styles should live in a separate feature branch, so I just wanted to clear this up. Any input? Thanks!

Reply:
No, only styles that are needed solely for the separate styling. Any styling to do with the zero-rated specific features (like terms of service, the user profile feature, telephone sign up, etc.) should not be in the styling branch.